src/screen/ApiLoadingScreen.tsx

```diff
@@ -45,31 +45,77 @@ export function ApiLoadingScreen({
         )}
       />
       <View style={globalStyles.centeredContainer}>
-        <View style={styles.textContainer}>
-          <Icon
-            style={[globalStyles.inlineIconDimension, {color: colorGreen}]}
-            name="planet"
-            pack="ionic"
-          />
-          <Text style={styles.text}>
-            Connecting to {currentNetwork?.name ?? ''}
-          </Text>
-        </View>
-        <ActivityIndicator size={'large'} color={colorGreen} />
+        {(() => {
```

Comment:
Don't understand why this is wrapped in a IIFE and the usage of `<>`

Reply:
well, I thought writing the conditions in if...else would be more readable than the ternary operator.
I could use a View with center alignment instead of <>, however, because the wrapping view was aligned already I used <> so the components instead would be direct children of that.

should I change this?

Reply:
Ah, I see. That mean, if you omit the `<>`, the alignment will break?

Reply:
you can't omit the `<>` [(React.Fragment)](https://reactjs.org/docs/fragments.html#short-syntax) because there are two children inside, that would be a JSX syntax error. I can replace it with a a `<View>` though but I have to style it so the alignment is right.

Reply:
True, didn't see that. haha